manifest-editor/components/organisms/CanvasView.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useEffect } from "react";
+import { useCanvas } from "@hyperion-framework/react-vault";
+
+import { CanvasContainer } from "../atoms/CanvasContainer";
+
+export const CanvasView: React.FC<{manifest: string}> = (manifest) => {
+  const viewer = useRef();
+  const canvas = useCanvas();
+  console.log(manifest.manifest);
+
+  // required for next js ssr
+  useEffect(() => {
+    import("@digirati/canvas-panel-web-components");
```

Comment:
Only temporary hopefully! There is a React component too which we can switch too later